frontend/src/pages/ranking/index.tsx

```diff
@@ -29,15 +29,31 @@ const COLORS = [
   },
 ];
 
-export const RankingPage: React.FC = () => {
+type Filters = {
+  topic: string;
+};
+
+export const RankingPage = () => {
   const conferenceCode = process.env.conferenceCode;
-  const { loading, data } = useRankingSubmissionQuery({
+
+  const {
+    data: {
+      conference: { topics },
+    },
+  } = useTopicsQuery({
+    variables: {
+      code: conferenceCode,
+    },
+  });
+  const [filters, { select }] = useFormState<Filters>({ topic: topics[0].id });
```

Comment:
What is the sorting? database dependant? or are topics ordered?

Reply:
Think database dependent, they arrive order by Id

Reply:
uhm oki, well that's ok for now I guess :)